README.md

```diff
@@ -0,0 +1,50 @@
+# Lab Setup and Containerization
+
+## Build Instructions
+The lab contains mulitple experiments which can all be built using the lab management repository.
+Follow the steps below to clone the necessary repositories, make required changes, and build the lab.
+
+1. Clone both the lab repository and the lab management repository:
+    ```
+    git clone https://github.com/virtual-labs/lab-image-processing-test-iiith.git
+    git clone https://github.com/virtual-labs/ph3-lab-mgmt.git
+    ```
+
+2. If required, make changes to the lab-descriptor.json in the lab repository to update the experiment repositories and their respective tags (version).
```

Comment:
Currently the lab only has 1 test experiment so let's mention that. When we build the main lab repo, we will specify this instruction